db/tests/conftest.py

```diff
@@ -37,6 +37,7 @@ def engine_with_roster(engine_with_schema):
 @pytest.fixture
 def engine_with_filter_sort(engine_with_schema):
     engine, schema = engine_with_schema
+    engine.dialect.ischema_names.update(types.CUSTOM_TYPE_DICT)
```

Comment:
Could you comment on why this is needed? I've been writing a lot of these fixture these past few days and some of the mechanics are still somewhat opaque to me.

Reply:
@dmos62 You need to either use a fixture that creates (or uses) an engine that includes our custom types, or you need to manually add the custom types to the engine for the custom types to be usable in filtering.  While `DATE` isn't a custom DB type, it _is_ a custom python type.